### PR TITLE
OCLOMRS-875 : strange concept on edit Concept page

### PR DESCRIPTION
**Pull request by nk183**
# JIRA TICKET NAME:
[JIRA OCLOMRS-875 ](https://issues.openmrs.org/browse/OCLOMRS-875 )

# Summary:
The string formatting on the edit Concept page is removed in this PR
this PR is an extension of https://github.com/openmrs/openmrs-ocl-client/pull/649

### CODE REVIEW

File: src/apps/concepts/pages/CreateOrEditConceptPage.tsx [hunk 1]
```diff
@@ -143,7 +143,7 @@ const CreateOrEditConceptPage: React.FC<Props> = ({
       title={
         context === CONTEXT.edit
           ? "Edit " +
-            startCase(toLower(concept ? concept.display_name : "concept"))
+            ( concept ? concept.display_name : "concept" )
```
Comment (HerbertYiga) [inline src/apps/concepts/pages/CreateOrEditConceptPage.tsx:146]: Did you try testing the changes with a working OCL instance ,cc @nk183

Comment (nk183) [reply src/apps/concepts/pages/CreateOrEditConceptPage.tsx:146]: Yes @HerbertYiga  it working fine
image

**Review by HerbertYiga — APPROVED** [review]
@nk183  thanks cc @dkayiwa

**Review by hadijahkyampeire — APPROVED** [review]
Thanks @nk183

**Pull request merged**